airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -60,15 +59,23 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
 
     def __init__(
         self,
-        connections_prefix: str = 'airflow/connections',
-        variables_prefix: str = 'airflow/variables',
-        profile_name: Optional[str] = None,
-        sep: str = "/",
+        connections_prefix=None,
+        variables_prefix=None,
```

Comment:
It also breaks backwards-compatibility

Reply:
No. It's doesn't break, because we always use keyword arguments.
https://airflow.readthedocs.io/en/latest/configurations-ref.html#backend-kwargs

Reply:
it does. If no-one passes anything, the default it used to take is `airflow/variables`

Reply:
Anyway, I don't see that these should be required arguments. In my current company, we share an AWS account with other departments. As so, we have to start our secrets with the prefix 'data_', hence in my current situation it does make sense to have `connections_prefix='data'`. However, in my previous company, we as BI team had an AWS account for ourselves, and thus we had the secrets just like _'db_redshift'_, _'api_salesforce'_ and so on, so the None value made more sense.

Furthermore, I would never expect to have the default variables as airflow/variables, because I won't ever think at all using the '/' sep in this case.

I know that what I think or believe is pointless, but I am wondering if there are more people with similar reasonings as me that the default values don't fit with their project.